package.json

```diff
@@ -16,138 +16,57 @@
   "engines": {
     "vscode": "^1.21.0"
   },
-  "categories": ["Formatters", "Programming Languages", "Linters", "Snippets"],
-  "keywords": ["ocaml", "reason", "bucklescript", "reasonml", "merlin"],
+  "categories": [
+    "Formatters",
+    "Programming Languages",
+    "Linters",
+    "Snippets"
+  ],
+  "keywords": [
+    "ocaml",
+    "reason",
+    "bucklescript",
+    "reasonml",
+    "merlin"
+  ],
   "icon": "assets/logo.png",
-  "activationEvents": ["onLanguage:ocaml", "onLanguage:reason"],
+  "activationEvents": [
+    "onLanguage:ocaml",
+    "onLanguage:reason"
+  ],
   "main": "./out/src/extension",
   "contributes": {
-    "commands": [
-      {
-        "command": "reason.caseSplit",
-        "title": "Reason: Case Split"
-      },
-      {
-        "command": "reason.showMerlinFiles",
-        "title": "Reason: Show Merlin Files"
-      },
-      {
-        "command": "reason.showAvailableLibraries",
-        "title": "Reason: Show Libraries Available via Dependencies"
-      },
-      {
-        "command": "reason.showProjectEnv",
-        "title": "Reason: Show Environment"
-      }
-    ],
     "configuration": {
       "type": "object",
       "title": "Reason configuration",
       "properties": {
-        "reason.codelens.unicode": {
-          "type": "boolean",
-          "default": true,
-          "description": "Enable the use of unicode symbols in codelens."
-        },
-        "reason.codelens.enabled": {
-          "type": "boolean",
-          "default": false,
-          "description": "Specifies whether the code lens is shown."
-        },
-        "reason.debounce.linter": {
-          "oneOf": [
-            {
-              "type": "integer"
-            },
-            {
-              "enum": ["Infinity"]
-            }
-          ],
-          "default": 500,
-          "description":
-            "How long to idle (in milliseconds) after keypresses before refreshing linter diagnostics. Smaller values refresh diagnostics more quickly."
-        },
-        "reason.diagnostics.tools": {
-          "type": "array",
-          "items": {
-            "enum": ["merlin", "bsb"]
-          },
-          "default": ["merlin"],
-          "maxItems": 2,
-          "uniqueItems": true,
-          "description":
-            "Specifies which tool or tools will be used to get diagnostics. If you choose both \"merlin\" and \"bsb\", merlin will be used while editing and bsb when saving."
-        },
         "reason.format.width": {
-          "type": ["number", null],
+          "type": [
+            "number",
+            null
+          ],
           "default": null,
           "description": "Set the width of lines when formatting code with refmt"
         },
-        "reason.path.bsb": {
-          "type": "string",
-          "default": "./node_modules/bs-platform/lib/bsb.exe",
-          "description": "The path to the `bsb` binary."
-        },
-        "reason.path.ocamlfind": {
```

Comment:
@jordwalke Heads up: this will remove the APIs for introspecting environments that were added in https://github.com/freebroccolo/ocaml-language-server/pull/68.